mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5299 net/minecraft/world/gen/feature/StructureSpacing
+	FIELD field_24647 STRUCTURE_SPACING Lcom/google/common/collect/ImmutableMap;
+	CLASS class_5300 StructureSpacingInformation
```

Comment:
Just `Information` should suffice as it's an inner class

Reply:
Fixed!